### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by simPod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,16 +66,16 @@ $config->setMetadataRefreshIntervalMs(10000);
 $config->setMetadataBrokerList('10.13.4.159:9192');
 $config->setBrokerVersion('0.9.0.1');
 $config->setRequiredAck(1);
-$config->setIsAsyn(false);
+$config->setIsAsyn(true);
```
Comment (lcobucci) [inline README.md]: I didn't manage to make this flag behave the way I understood it should, so I'd suggest to keep the way it was

File: README.md [hunk 2]
```diff
@@ -63,16 +66,16 @@ $config->setMetadataRefreshIntervalMs(10000);
 $config->setMetadataBrokerList('10.13.4.159:9192');
 $config->setBrokerVersion('0.9.0.1');
```
Comment (lcobucci) [inline README.md]: Can you please update this to `1.0.0`?

File: README.md [hunk 3]
```diff
@@ -63,16 +66,16 @@ $config->setMetadataRefreshIntervalMs(10000);
...
 $config->setRequiredAck(1);
-$config->setIsAsyn(false);
+$config->setIsAsyn(true);
 $config->setProduceInterval(500);
 $producer = new \Kafka\Producer(function() {
```
Comment (lcobucci) [inline README.md]: Can you please fix the indentation over here? Like:

```php
$producer = new \Kafka\Producer(
    function (): array {
        return [
            [
                'topic' => 'test',
                'value' => 'test....message.',
                'key'   => 'testkey',
            ],
        ];
    }
);
```

File: README.md [hunk 4]
```diff
@@ -137,7 +139,7 @@ $config->setMetadataRefreshIntervalMs(10000);
 $config->setMetadataBrokerList('10.13.4.159:9192');
 $config->setGroupId('test');
 $config->setBrokerVersion('0.9.0.1');
```
Comment (lcobucci) [inline README.md]: Let's update to `1.0.0` here too

File: README.md (hunk 1 above)
Comment (simPod) [reply README.md]: reverting to false then or should I remove it completely?

Comment (lcobucci) [reply README.md]: let's revert for now 😄

File: README.md [hunk 5]
```diff
@@ -61,19 +64,21 @@ $logger->pushHandler(new StdoutHandler());
...
-$producer = new \Kafka\Producer(function() {
-	return array(
-		array(
-			'topic' => 'test',
-			'value' => 'test....message.',
-			'key' => 'testkey',
-			),
-	);
-});
+$producer = new \Kafka\Producer(
```
Comment (lcobucci) [inline README.md]: It seems we still have some tabs here...

Comment (simPod) [reply README.md]: daym

**Comment by lcobucci** [comment]
#174 should fix the build =)

**Comment by lcobucci** [comment]
@simPod 🚢 thanks!

**Pull request merged**